mw-webapp/src/component/dropdown/Dropdown.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "src/styles/variables";
+
+$dropdownZIndex: 100;
```

Comment:
move Z index to global variables. It is pretty handy to see all zIndexes in one place

Reply:
done